_posts/2023-06-13-12.0.1-release.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Apache Arrow 12.0.1 Release"
+date: "2023-06-13 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 12.0.1 release.
+This is mostly a bugfix release that includes [**38 resolved issues**][1]
+from [**12 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## C++ notes
+
+
+## Go notes
```

Comment:
@zeroshade Can you add some notes. The list of changes from the changelog is:
```
    GH-34784 - [Go] Fix 32-bit build (#35767)
    GH-35337 - [Go] ASAN tests fail with Go1.20+ (#35338)
    GH-34657 - [Go] Add ValueString(i int) string to array (#34986)
```

Reply:
```suggestion

* Fixed builds of the Go Arrow package on 32-bit systems ([GH-34784](https://github.com/apache/arrow/pull/35767))
* Added `ValueString(int) string` method to `arrow.Array` ([GH-34657](https://github.com/apache/arrow/pull/34986))
* Fixed ASAN failure when using go1.20+ by using `unsafe.StringData` ([GH-35337](https://github.com/apache/arrow/pull/35338))
```